README.md

```diff
@@ -270,6 +270,28 @@ To shut down the development server, use
 ./develop_server stop
 ```
 
+## Heroku Review Apps ##
```

Comment:
Since this is used for testing I think it would make sense to put it in the testing section.